appinventor/components/src/com/google/appinventor/components/runtime/Web.java

```diff
@@ -122,6 +124,21 @@
     }
   }
 
+  /**
```

Comment:
It's possible we might want to reuse this exception for other classes that do network operations. Let's move it into `com.google.appinventor.components.runtime.errors`

Reply:
Moved it into it's own class.